### PR TITLE
Repair prescriptonImportComponent and in referrals add displaying refferals

### PR DESCRIPTION
**Pull request by glanttz**
- repair prescriptionComponentImport (you dont need to check it, only one variable was changed)
- add in main icon
- add  in referralsview component
- add displaying referrals from adding manually.
- please check displaying referrals (te co sa dodane za pomoca dodaj skierowanie recznie)

### CODE REVIEW

File: src/components/prescriptions/ImportGroupComponent.vue [hunk 1]
```diff
@@ -12,8 +12,8 @@
             <prescription-import-component
                 v-for="prescription in getPatientImportPrescriptions"
                 :key="prescription.id"
```
Comment (PixelSculptor) [inline src/components/prescriptions/ImportGroupComponent.vue:14]: github shows changes but I cannot spot it. Did u moved line below only?

File: src/components/prescriptions/PrescritionImport.vue [hunk 2]
```diff
@@ -5,9 +5,9 @@
         <h3 class="prescriptionBox__header">Dodaj receptę:</h3>
         <form class="prescriptionBox__form prescriptionForm">
             <div class="inputBox">
-                <label for="doctorData"> Lekarz:</label>
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescritionImport.vue]: Delete space before Lekarz: and add after. Or delete befroe and after and add margin-right to label

File: src/components/prescriptions/PrescritionImport.vue [hunk 3]
```diff
@@ -148,9 +148,7 @@
             isLoading.value = false;
             successFlag.value = true;
         } catch (err) {
-            console.log(err);
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescritionImport.vue]: besos :*

File: src/components/referrals/ImportReferralsComponent.vue [hunk 4]
```diff
@@ -112,7 +112,8 @@
         } finally {
             testFile.value = null;
             testFileError.value = null;
-            nameAndSurnameDoctor.value.researchDate.value = null;
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsComponent.vue:115]: what was it? i havent spotted it before. Youve divided object yes?

Comment (glanttz) [reply src/components/referrals/ImportReferralsComponent.vue:115]: noo, its missmatch. it need to be 
 nameAndSurnameDoctor.value= null;
 researchDate.value = null;

damn it soryy

File: src/components/prescriptions/ImportGroupComponent.vue (hunk 1 above)
Comment (glanttz) [reply src/components/prescriptions/ImportGroupComponent.vue:14]: yes sir

File: src/components/referrals/ReferralsComponentLabel.vue [hunk 5]
```diff
@@ -0,0 +1,104 @@
+<template>
+    <li class="documentBox">
+        <font-awesome-icon :icon="classOfIcon" />
+        <h5 class="documentBox__title">{{ name }} {{ date }}</h5>
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponentLabel.vue]: Divide this into 2 tags. Name should be in h5 and date in different tag (not wrapped in h5)

File: src/components/referrals/ReferralsComponentLabel.vue [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+            type: String,
+            default: '',
+        },
+        id: {
+            type: String,
+            default: null,
+        },
+        typeOfDoc: {
+            type: String,
+            default: 'referralView',
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponentLabel.vue]: What does mean this default value?

File: src/components/referrals/ReferralsComponentLabel.vue [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+</script>
+
+<style lang="scss" scoped>
+    .documentBox {
+        width: 100%;
+        border: $border-size solid $gray-300;
+        border-radius: $border-radius--normal;
+        padding: 1rem;
+        background-color: $white;
+        @include flex-position(row, nowrap, space-between, center);
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponentLabel.vue]: move import this mixin to the top of selector (to 88 line)

File: src/components/referrals/ReferralsDetails.vue [hunk 8]
```diff
@@ -0,0 +1,83 @@
+<template>
+    <section class="detailsReferrence">
+        <h3 class="detailsReferrence__header">Szczegóły skierowania</h3>
+        <form class="detailsReferrence__form detailsReferrence">
+            <div class="inputBox">
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsDetails.vue:5]: Here is good structure. Similar have to be added to ReferralsComponentLabel.vue

File: src/components/referrals/ReferralsDetails.vue [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+            height: 85%;
+            .inputBox {
+                @include flex-position(row, nowrap, space-between, center);
+                width: 100%;
+                h5 {
+                    @include label;
+                    color: $blue-900;
+                }
+            }
+            data {
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsDetails.vue]: it doesnt work. You havent any html element with class 'data'. Only you have detailsReferrence__data. So you have to change on this name or something like this: &__data but in this case selector should be hoisted in structure up.

File: src/components/referrals/ReferralsGroupComponent.vue [hunk 10]
```diff
@@ -0,0 +1,78 @@
+<template>
+    <section class="referralsContainer">
+        <h3 class="referralsContainer__header">
+            Dodane skierowania
+            <span class="numOfPres">{{ getPatientRefferals?.length }}</span>
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsGroupComponent.vue]: change to numOfRef or sth like this

File: src/components/referrals/ReferralsGroupComponent.vue [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+<script setup>
+    import { storeToRefs } from 'pinia';
+    import { onMounted, ref } from 'vue';
+
+    import useUserStore from '../../stores/userStore';
+    import FallbackInfoComponent from '../FallbackInfoComponent.vue';
+    import ReferralsComponentLabel from './ReferralsComponentLabel.vue';
+
+    const testInfo = ref('Nie dodano żadnego skierowania');
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsGroupComponent.vue]: change var name to refferalsInfo casue its not this view

File: src/components/referrals/ReferralsGroupComponent.vue [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+    const userStore = useUserStore();
+    const { getPatientRefferals } = storeToRefs(userStore);
+
+    onMounted(async () => {
+        await userStore.fetchPatientReferrals();
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsGroupComponent.vue]: better idea is to fetch data in view not in nested component

File: src/components/referrals/ImportReferralsComponent.vue (hunk 4 above)
Comment (PixelSculptor) [reply src/components/referrals/ImportReferralsComponent.vue:115]: luz I had to be sured ;) no offense

File: src/components/prescriptions/PrescritionImport.vue (hunk 2 above)
Comment (glanttz) [reply src/components/prescriptions/PrescritionImport.vue]: i delete before and add after 🗡️

File: src/components/referrals/ReferralsComponentLabel.vue (hunk 6 above)
Comment (glanttz) [reply src/components/referrals/ReferralsComponentLabel.vue]: now i thing that all typeOfDoc is unnecesarry

Comment (glanttz) [reply src/components/referrals/ReferralsComponentLabel.vue]: i dont use it, so i will delete it.

File: src/components/prescriptions/PrescriptionComponent.vue [hunk 13]
```diff
@@ -1,7 +1,8 @@
 <template>
     <li class="documentBox">
         <font-awesome-icon :icon="classOfIcon" />
-        <h5 class="documentBox__title">{{ name }} {{ date }}</h5>
+        <h5 class="documentBox__title">{{ name }}</h5>
+        <p class="documentBox__title">{{ date }}</p>
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionComponent.vue]: Is it expected? Youve got two element (other semantic) with same class.  You want to style them with the same selector (they will look both same).

File: src/components/prescriptions/PrescriptionComponent.vue [hunk 14]
```diff
@@ -43,10 +44,11 @@
...
     const modalDetils = ref(null);
-    onClickOutside(modalDetils, () => (openModalDetails.value = false));
     const userStore = useUserStore();
     const { getPatientPrescriptions } = storeToRefs(userStore);
 
+    onClickOutside(modalDetils, () => (openModalDetails.value = false));
+
     defineProps({
         name: {
             type: String,
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionComponent.vue:47]: move define props to the top of file but below imports

File: src/components/prescriptions/PrescritionImport.vue [hunk 15]
```diff
@@ -5,7 +5,7 @@
         <h3 class="prescriptionBox__header">Dodaj receptę:</h3>
         <form class="prescriptionBox__form prescriptionForm">
             <div class="inputBox">
-                <label for="doctorDataImport"> Lekarz:</label>
+                <label for="doctorDataImport">Lekarz: </label>
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescritionImport.vue]: checked

File: src/components/prescriptions/PrescritionImport.vue [hunk 16]
```diff
@@ -19,7 +19,7 @@
             </div>
 
             <div class="inputBox">
-                <label for="medicine">Lek:</label>
+                <label for="medicine">Lek: </label>
                 <select
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescritionImport.vue]: checked

File: src/components/prescriptions/PrescritionImport.vue [hunk 17]
```diff
@@ -33,7 +33,7 @@
             </div>
 
             <div class="inputBox">
-                <label for="frequencys">Częstotliwość:</label>
+                <label for="frequencys">Częstotliwość: </label>
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescritionImport.vue]: checked

Comment (PixelSculptor) [reply src/components/prescriptions/PrescritionImport.vue]: but btw its frequency (single) and frequencies (plural) ;)

File: src/components/referrals/ImportReferralsGroupComponent.vue [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+        <h3 class="referralsContainer__header">
+            Zaimportowane skierowania
+            <span class="numOfImport">{{
+                getPatientImportReferrals?.length
+            }}</span>
+        </h3>
+        <ul
+            v-if="getPatientImportReferrals?.length"
+            class="referralsContainer__list">
+            <import-referrals-labels
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsGroupComponent.vue:12]: for the future: good naming for v-for single element is name with single noun (e.g. refferal)

File: src/components/referrals/ImportReferralsGroupComponent.vue [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+    const { getPatientImportReferrals } = storeToRefs(userStore);
+
+    const importInfo = ref('Nie zaimportowano żadnych skierowań');
+</script>
+
+<style lang="scss" scoped>
+    .referralsContainer {
+        width: 100%;
+        height: 25vh;
+        @include flex-position(column, nowrap, flex-start, flex-start);
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsGroupComponent.vue]: include to top

File: src/components/referrals/ImportReferralsGroupComponent.vue [hunk 20]
```diff
@@ -0,0 +1,72 @@
...
+        gap: 0.1rem;
+        &__header {
+            @include text-header3($font-weight-semiBold);
+            color: $blue-900;
+            @include flex-position(row, nowrap, flex-start, center);
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsGroupComponent.vue:49]: same

File: src/components/referrals/ReferralsComponentLabel.vue [hunk 21]
```diff
@@ -1,7 +1,8 @@
 <template>
     <li class="documentBox">
         <font-awesome-icon :icon="classOfIcon" />
-        <h5 class="documentBox__title">{{ name }} {{ date }}</h5>
+        <h5 class="documentBox__title">{{ name }}</h5>
+        <p class="documentBox__title">{{ date }}</p>
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponentLabel.vue]: same class? as above

File: src/components/referrals/ImportReferralsLabels.vue [hunk 22]
```diff
@@ -0,0 +1,65 @@
...
+</script>
+
+<style lang="scss" scoped>
+    .documentBox {
+        width: 100%;
+        border: $border-size solid $gray-300;
+        border-radius: $border-radius--normal;
+        padding: 1rem;
+        background-color: $white;
+        @include flex-position(row, nowrap, space-around, center);
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsLabels.vue]: include

File: src/components/prescriptions/PrescriptionComponent.vue (hunk 13 above)
Comment (glanttz) [reply src/components/prescriptions/PrescriptionComponent.vue]: change style from paragraph to new style data:
&__data {
            @include text-paragraph($font-weight-light);
            color: $gray-700;
        }

File: src/components/prescriptions/PrescriptionComponent.vue (hunk 14 above)
Comment (PixelSculptor) [reply src/components/prescriptions/PrescriptionComponent.vue:47]: Okay but calling function should be below vars not between them

File: src/components/prescriptions/PrescriptionComponent.vue (hunk 13 above)
Comment (PixelSculptor) [reply src/components/prescriptions/PrescriptionComponent.vue]: ok gut

**Review by PixelSculptor — COMMENTED** [review]
move this funtion call below then approve and check my pr

**Review by PixelSculptor — APPROVED** [review]
Do this fast fix and merge youself

**Pull request merged**